api/migrations/20210525230253-create-reminder.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("reminders", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal("uuid_generate_v4()"),
+      },
+      channel: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      userId: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        references: {
+          model: "users",
+          key: "id",
+        },
+      },
+      body: {
+        allowNull: false,
+        type: Sequelize.TEXT,
```

Comment:
Will there be limit on the body size?

Reply:
Yes, but that is something that can be limited in the frontend.

Reply:
By frontend you mean the bot logic?